app.js

```diff
@@ -0,0 +1,111 @@
+// Onboarding Project - 2018
+var _this = this;
```

Comment:
Just something to keep in mind when you're setting up a project from scratch: don't commit generated files or build artifacts because devs will generate their own when they build the project. Typically these artifacts are added to the `.gitignore` file and won't show up when you run a `git commit`.

Reply:
apologies, realized this after commit but didn't know how to remove already committed files,  resolved